scripts/scripts.js

```diff
@@ -98,19 +98,32 @@ export async function decorateMain(main) {
   decorateVideoLinks(main);
 }
 
+/**
+ * Add three entries to the head, in this order:
+ * <!-- Load SAP theme (here: sap_glow) -->
+ * <link rel="stylesheet" href="/themes/sap_glow/css_variables.css">
+ * <!-- Announce theme to UI5 components (before module is loaded) -->
+ * <script data-ui5-config type="application/json">{"theme": "sap_glow"}</script>
+ * <!-- Load SAP Digital Design System Web Components Module -->
+ * <script src="/libs/dds-wc-bundle.esm.m.js" type="module"></script>
+ * @returns {Promise<void>}
```

Comment:
This is waste of bytes send to the user, the code below is clear and simple.

Reply:
Removed.